pkg/rules/packages-to-test.yml

```diff
@@ -1,34 +0,0 @@
----
```

Comment:
So, I presume testing is now done in a different way?

Reply:
No, it's just that that file was a complete duplicate of the build instructions, newer Ploutos doesn't require you to manually specify the same stuff twice if not needed.

Reply:
should have read your "Changes" above better.. sorry